common/journal_entries/dvg_the_cone.txt

```diff
@@ -0,0 +1,33 @@
+﻿je_the_cone = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+
+	is_shown_when_inactive = {
+	    OR = {
+			country_has_primary_culture = cu:quechua
+			country_has_primary_culture = cu:beornian 
+		}
+    }
+
+	possible = {
+		OR = {
+			has_law = law_type:law_colonial_exploitation
+			has_law = law_type:law_colonial_resettlement
+		}
+        OR = {		
+		     owns_entire_state_region = STATE_NEW_CORNWALL
+		     owns_entire_state_region = STATE_COLLASUYU
+		}
+    }
+    
+
+	complete = {
+			#DVG
+			owns_entire_state_region = STATE_LANDS_END
+		}
```

Comment:
NIT: Can you fix the formatting? And duplicate new lines.
![image](https://user-images.githubusercontent.com/116951759/199550896-2f96e0cc-ac77-4d2b-9667-6dbd1afa7e49.png)

Reply:
![image](https://user-images.githubusercontent.com/116951759/199557892-f55e687a-dc65-4f9c-b71c-a10c1fd84678.png)
Need to move 1 tab back on owns+ and then its fine.